specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/virtualMachines.json

```diff
@@ -142,14 +142,9 @@
     },
     "HostInfoListResult": {
       "description": "Result of the request to list cluster hosts",
-      "properties": {
-        "value": {
```

Comment:
was this a swagger mistake or is the API actually changing?

Reply:
> was this a swagger mistake or is the API actually changing?

It was swagger mistake. Thanks.